includes/class-wc-payments-account.php

```diff
@@ -539,6 +540,16 @@ private function get_oauth_return_url( $wcpay_connect_from ) {
 	 * @param string $wcpay_connect_from - where the user should be returned to after connecting.
 	 */
 	private function init_stripe_oauth( $wcpay_connect_from ) {
+		if ( get_transient( self::ON_BOARDING_STARTED_TRANSIENT ) ) {
+			$this->redirect_to_onboarding_page(
+				__( 'There was a duplicate attempt to redirect you to Stripe. Please wait a few seconds and try again.', 'woocommerce-payments' )
```

Comment:
We don't explicitly mention the redirect to Stripe before this, so we might want to just generalize that to "duplicate attempt to initiate account setup" or something like that.